src/Data/Nat/Properties.agda

```diff
@@ -1848,7 +1848,7 @@ m∸n≤∣m-n∣ m n with ≤-total m n
   n ∸ m     ∎
 
 *-distribˡ-∣-∣ : _*_ DistributesOverˡ ∣_-_∣
-*-distribˡ-∣-∣ a = wlog ≤-total (comm⇒sym[distribˡ] {_◦_ = _*_} ∣-∣-comm a)
+*-distribˡ-∣-∣ a = wlog ≤-total (comm⇒sym[distribˡ] ∣-∣-comm {_◦_ = _*_} a)
```

Comment:
This makes this look like not a cosmetic change but a breaking change?

Reply:
Sigh.
The proof above was itself only introduced last week in #2577 ... and I guess that it is inevitable that the move to using `variable` has the potential *always* to reorder the order of *implicit* arguments in proofs. In the past we haven't (apparently?) regarded this as `breaking` behaviour, possible because the 'real' underlying issue may be that the ambient parameterisation on the binary operation ought to be explicit. Will investigate whether there's a better fix to the above.

Reply:
You're probably right it should be v3.0, and then we can fix the parametrisation problems exposed above as well.

Reply:
... and I'd press on the argument in favour of #2572 (backwards compatible!) as well as part of such v3.0 changes...